legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -116,7 +116,7 @@ def validate_roles(incorporation_json) -> Error:
         err_path = '/filing/incorporationApplication/parties/roles'
         msg.append({'error': 'Must have a maximum of one completing party', 'path': err_path})
 
-    if incorporation_json['filing']['incorporationApplication']['nameRequest']['legalType'] == 'BC':
+    if incorporation_json['filing']['business'].get('legalType','BC') == 'BC':
```

Comment:
I think this line is incorrect according to the schema. You won't have a business section in an incorp filing -> legalType will be in the namerequest section

Reply:
Yes we do.

![image](https://user-images.githubusercontent.com/10002889/83662101-f056c400-a57b-11ea-91ba-5e4b5b4bb8b1.png)

Reply:
But a Numbered Company IA filing does not have an incorporationApplication object.

![image](https://user-images.githubusercontent.com/10002889/83662261-209e6280-a57c-11ea-9618-c4597cf8f162.png)

Reply:
Interesting okay well then we should change the logic in the schema: right now it says if not incorp then business is required. Now we're depending on the business to be there in an incorp so it should be required for incorp as well

Reply:
Does it make sense to update the schema in THIS ticket?